apps/fluent-tester/src/TestComponents/RadioGroupV1/CustomizedRadioGroup.tsx

```diff
@@ -61,7 +61,7 @@ export const CustomizedRadioGroup: React.FunctionComponent = () => {
       radioBorderWidth,
       radioVisibility,
       radioBorder,
-      radioOuterCircleSize,
+      // radioOuterCircleSize,
```

Comment:
Is this meant to be commented out?

Reply:
This was initially preventing me from building for whatever reason, forgot to uncomment this out.